.travis.yml

```diff
@@ -151,6 +151,47 @@ jobs:
         - EXTRA_CXXFLAGS="-DDEBUG"
       script: echo "Not running any tests for a debug build."
 
+    # Run Coverity
+    - stage: Test different OS/CXX/Flags
+      os: linux
+      sudo: required
+      compiler: gcc
+      cache: ccache
+      addons:
+        apt:
+          sources:
+            - ubuntu-toolchain-r-test
+          packages:
+            - libwww-perl
+            - g++-5
+            - libubsan0
+        coverity_scan:
+          project:
+            name: "diffblue/cbmc"
+            description: "Travis build of ${TRAVIS_COMMIT}"
+          notification_email: "coverity-scan@diffblue.com"
+          build_command_prepend: "make -C src minisat2-download"
+          build_command: "make -C src -j2"
+          branch_pattern: "develop"
+      before_install:
+        - |
+          if [[ "${TRAVIS_EVENT_TYPE}" != "cron" ]]
+          then
+            echo "This is not a cron build and build is not needed."
+            travis_terminate 0
+          fi
+        - mkdir bin ; ln -s /usr/bin/gcc-5 bin/gcc
+        - echo -n | openssl s_client -connect scan.coverity.com:443 |
+          sed -ne '/-BEGIN CERTIFICATE-/,/-END CERTIFICATE-/p' |
+          sudo tee -a /etc/ssl/certs/ca-certificates.crt
```

Comment:
Why is this necessary? trusty does ship the required CA certificate.

Reply:
I was recommended by coverity, but I will try it without it.